src/modules/tenant/service.js

```diff
@@ -7,6 +7,20 @@ export default class TenantService extends Service {
 		super(app, name);
 	}
 
+	async initialize() {
+
+		// If the tenant list is in the configuration, use it
```

Comment:
We would need to extend the documentation about tenant usage without authorization
https://github.com/TeskaLabs/asab-webui/blob/implement/add-tenants-config/doc/tenant.md

Reply:
Documentation extended.